src/pages/productpage/finalProducts/finalProducts.js

```diff
@@ -0,0 +1,46 @@
+
+import {AllProducts} from "../allproducts/AllProducts"
+
+import "../allproducts/allproducts.css"
+
+
```

Comment:
-use code formatter and remove unnecessary space between two lines of code 
-i think it is enough to import components and need not import css related to the  component when your importing the components as you would have already imported them in the repective components file